app/src/main/res/values/dimens.xml

```diff
@@ -15,6 +15,7 @@
   <dimen name="elevation_0dp">0dp</dimen>
   <dimen name="padding_72dp">72dp</dimen>
   <dimen name="padding_24dp">24dp</dimen>
+  <dimen name="margin_24dp">24dp</dimen>
```

Comment:
Instead of introducing these padding here, prefer directly hard-coding them where ever you are using them

Reply:
Done